examples/sync-entrateam-accesslist-eks-access/README.md

```diff
@@ -0,0 +1,57 @@
+# Create Access List which allows, users of an Entra Group, access to discovered EKS Clusters
+
+This program allows users from a given Entra Group to access all auto discovered EKS Clusters for a given AWS account ID.
+
+It does the following:
+ - create a Teleport Role which allows access to auto discovered EKS Clusters for a given account ID, which is inherited from a trait
+ - create an Access List per account ID that:
+   - members get access to the role above
+   - injects the AWS account ID as a trait
+   - adds another Access List as a sub-list/member, which must be a Entra Group
+
+You'll have a single Teleport Role, but as many Access Lists as Entra Groups.
+
+### Running the script 
+
+1. Ensure your Teleport user has the minimum required RBAC permissions to perform the tasks run by the script.
+   
+   The following Teleport RBAC rules are required:
+   ```yaml
+       - resources:
+         - roles
+         verbs:
+         - read
+         - list
+         - create
+       - resources:
+         - kube_server
+         verbs:
+         - read
+         - list
+       - resources:
+         - access_list
+         verbs:
+         - read
+         - list
+         - create
+         - update
+   ```
+2. Authenticate to your Teleport Control Plane via 'tsh login'.
+3. Update the Teleport tenant name in the main.go file to your Teleport tenant name.
+4. Run the Access List Go script. The example below uses -aws-account-id along with -group-by-name. However, you can provide the Entra ID Group using one of the following flag options:
+
+  - the Teleport's Access List ID as seen in Teleport
```

Comment:
@marcoandredinis could you list to specific flag options that can be passed via command?